{
  "channeldata": {
    "activate.d": false,
    "binary_prefix": false,
    "deactivate.d": false,
    "description": null,
    "dev_url": "https://github.com/LuminosoInsight/python-ftfy",
    "doc_source_url": null,
    "doc_url": "http://ftfy.readthedocs.io/",
    "home": "https://github.com/LuminosoInsight/python-ftfy/",
    "icon_hash": null,
    "icon_url": null,
    "identifiers": null,
    "keywords": null,
    "license": "MIT",
    "post_link": false,
    "pre_link": false,
    "pre_unlink": false,
    "recipe_origin": null,
    "run_exports": {},
    "source_git_url": null,
    "source_url": "https://pypi.io/packages/source/f/ftfy/ftfy-4.4.2.tar.gz",
    "subdirs": [
      "win-64"
    ],
    "summary": "Fixes some problems with Unicode text after the fact",
    "tags": null,
    "text_prefix": false,
    "timestamp": 0,
    "version": "4.4.2"
  },
  "channeldata_version": 1,
  "feedstock": null,
  "labels": [
    "main",
    "cf202003",
    "cf201901"
  ],
  "package": "ftfy-4.4.2-py36_0.tar.bz2",
  "repodata": {
    "build": "py36_0",
    "build_number": 0,
    "depends": [
      "html5lib",
      "python 3.6*",
      "setuptools",
      "wcwidth"
    ],
    "license": "MIT",
    "md5": "f66b73e8a0ffa781322875acd9229fa0",
    "name": "ftfy",
    "sha256": "f6d00c83aadde0f3f69182ac7ba0ea6c2b26c66f1e13b6698e6af99f86390ba5",
    "size": 94001,
    "subdir": "win-64",
    "version": "4.4.2"
  },
  "repodata_version": 1,
  "subdir": "win-64",
  "url": "https://conda.anaconda.org/conda-forge/win-64/ftfy-4.4.2-py36_0.tar.bz2"
}